File: 2019-10-16/Iphone7Test/Iphone7Test/ViewController.m
//
//  ViewController.m
//  Iphone7Test
//
//  Created by 吉腾蛟 on 2019/10/17.
//  Copyright © 2019 JY. All rights reserved.
//

#import "ViewController.h"
#import "CustomAlertView.h"

#define SCREEN_WIDTH [UIScreen mainScreen].bounds.size.width
#define SCREEN_HEIGHT [UIScreen mainScreen].bounds.size.height

@interface ViewController ()
@property(nonatomic,strong)CustomAlertView *alertView;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}
-(CustomAlertView *)alertView{
    if (!_alertView) {
        _alertView=[[CustomAlertView alloc] initWithFrame:CGRectMake(0, 0, SCREEN_WIDTH, SCREEN_HEIGHT)];
//        _alertView.delegate=self;
    }
    return _alertView;
}
- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event{
    [[UIApplication sharedApplication].keyWindow addSubview:self.alertView];
}

@end

File: 2019-5-29(SWift)/Swipes/Swipes/ViewController.swift
//
//  ViewController.swift
//  Swipes
//
//  Created by 吉腾蛟 on 2019/9/25.
//  Copyright © 2019 JY. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet var label: UILabel!
    private var gestureStartPoint: CGPoint!
    
    private let minimumGestureLength = Float(25.0)
    private let maximumVariance = Float(5)
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        let touch = touches.randomElement()!
        gestureStartPoint = touch.location(in: self.view)
    }
    
    override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent?) {
        let touch = touches.randomElement()!
        let currentPosition = touch.location(in: self.view)
        
        
    }
}

